fcrepo-client-impl/src/main/java/org/fcrepo/client/impl/FedoraDatastreamImpl.java

```diff
@@ -93,6 +93,11 @@ public void setGraph( final Graph graph ) {
     }
 
     @Override
+    public String getPropertiesPath() {
+        return path + "/fcr:metadata";
```

Comment:
The HTML client uses the `describedBy` header from the response to the resource itself. That is the LDP-way.
This `/fcr:metadata` approach will work for now, but is not the ideal solution.
...but fcrepo4-client is not aligned with LDP at this point... which is another topic altogether.